docs/source/bibliography.md

```diff
@@ -26,7 +26,7 @@ order:
 
 ## Software
 
-poliastro wouldn\'t be possible without the tremendous, often unpaid and
+poliastro wouldn\'t have been possible without the tremendous, often unpaid and
```

Comment:
`poliastro wouldn't be` is correct here 😄

Reply:
Thanks for your suggestion @xlxs4. I have reverted that change.